assignments/assignment_1/README.md

```diff
@@ -33,11 +46,14 @@ For this refactoring section, we have 3 activities:
 
 Now, let's create a CI pipeline for this project. We will use GitHub Actions for this.
 
-1. Create a new repo for this project.
+1. Create a new repo for this project on GitHub.
```

Comment:
The idea is to create a personal public repo, right?

Reply:
Yes!